packages/cosmwasm/src/multisigcosmwasmclient.spec.ts

```diff
@@ -0,0 +1,202 @@
+      pendingWithoutLaunchpad();
+      pendingWithoutCw3();
+      const contractAddress = deployedCw3.instances[0];
+      const wallet = await Secp256k1HdWallet.fromMnemonic(alice.mnemonic);
+      const client = new MultisigCosmWasmClient(launchpad.endpoint, alice.address0, wallet, contractAddress);
+      const toAddress = makeRandomAddress();
+      const msg = {
+        bank: {
+          send: {
+            from_address: contractAddress,
+            to_address: toAddress,
+            amount: [
+              {
+                amount: "1",
+                denom: "ucosm",
+              },
+            ],
+          },
+        },
+      };
+      const { logs } = await client.createMultisigProposal(
+        "My proposal",
+        "A proposal to propose proposing proposals",
+        [msg],
+      );
+      const wasmEvents = logs[0].events.find((event) => event.type === "wasm");
+      assert(wasmEvents, "Wasm events not found in logs");
+      const proposalIdAttribute = wasmEvents.attributes.find((log) => log.key === "proposal_id");
+      assert(proposalIdAttribute, "Proposal ID not found in logs");
+      const proposalId = parseInt(proposalIdAttribute.value, 10);
+
+      const executeResult = await client.executeMultisigProposal(proposalId);
+      expect(executeResult).toBeTruthy();
+    });
+
+    it("proposal is accepted (proposer does not have enough weight alone)", async () => {
+      pendingWithoutLaunchpad();
+      pendingWithoutCw3();
+      const contractAddress = deployedCw3.instances[1];
+      const proposerWallet = await Secp256k1HdWallet.fromMnemonic(alice.mnemonic);
+      const proposer = new MultisigCosmWasmClient(
+        launchpad.endpoint,
+        alice.address0,
+        proposerWallet,
+        contractAddress,
+      );
+      const voterWallet = await Secp256k1HdWallet.fromMnemonic(alice.mnemonic, makeCosmoshubPath(1));
+      const voter = new MultisigCosmWasmClient(
+        launchpad.endpoint,
+        alice.address1,
+        voterWallet,
+        contractAddress,
+      );
+      const toAddress = makeRandomAddress();
+      const msg = {
+        bank: {
+          send: {
+            from_address: contractAddress,
+            to_address: toAddress,
+            amount: [
+              {
+                amount: "1",
+                denom: "ucosm",
+              },
+            ],
+          },
+        },
+      };
+      const { logs } = await proposer.createMultisigProposal(
+        "My proposal",
+        "A proposal to propose proposing proposals",
+        [msg],
+      );
+      const wasmEvents = logs[0].events.find((event) => event.type === "wasm");
+      assert(wasmEvents, "Wasm events not found in logs");
+      const proposalIdAttribute = wasmEvents.attributes.find((log) => log.key === "proposal_id");
+      assert(proposalIdAttribute, "Proposal ID not found in logs");
+      const proposalId = parseInt(proposalIdAttribute.value, 10);
```

Comment:
This requires the proposer to manually inform the voter about the proposal ID to vote on.

Could the voter get the open proposals from chain?